README.md

```diff
@@ -67,6 +67,16 @@ let player = Player(options: options)
 player.attachTo(yourView, controller: self)
 ```
 
+#### Media Control
+
+It controls all the user interaction between UI and the playback, it relies only on playback abstraction, therefore, this component should work in playback.
+
+It's a set of `Stack Views` divided into panels (`Top`, `Center` and `Bottom`) and can be used to organize the components on the screen.
```

Comment:
I believe we can mention how the `MediaControl` works instead of talking about its implementation. What do you think?

Some key points that we can mention:

1) It's an element that contains plugins which control the playback;
2) You can customize it with your own plugins (aka `Media Control Plugins`)
3) It is divided in panels (like you said)

Reply:
Done!